### PR TITLE
"Light & Dark Mode"

### LINKED ISSUE #4: Implement Light/Dark Mode Toggle Feature
Integrate a toggle button into the TicTacToe application to allow users to effortlessly switch between light and dark modes. This feature aims to enhance user accessibility and cater to individual preferences regarding the app's visual presentation. Ensure a smooth transition between modes.

### PR DESCRIPTION
**Pull request by rea-fernandes**
1. Added Games.css to add style to the game.
2. Importing this css in Game.js
3. Added Mode.css to support Dark and Light modes.
4. Importing this css in App.js
5. Added code to switch between the two modes.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,13 +1,28 @@
...
+        <div className="switch-checkbox">
+          <label className="switch">
+            <input type="checkbox" onChange={() => setDarkMode(!darkMode)} />
+            <span className="slider round"> </span>
+          </label>
+        </div>
+        <span style={{ color: darkMode ? "#c96dfd" : "grey" }}>☽</span>
+      </div>
+      <div>
+        <h1 className="welcome">{wel}</h1>
```
Comment (MuhammadAwaisGondal) [inline src/App.js]: Welcome message could be added directly like this` <h1 className="welcome">Welcome to TicTacToe!</h1>` as this message is not changing

File: src/App.js [hunk 2]
```diff
@@ -1,13 +1,28 @@
...
   let wel = "Welcome to TicTacToe!";
+  const [darkMode, setDarkMode] = useState(false);
   return (
-    <>
-    <div className="wel">{wel}</div>
-    <Game />
-    </>
+    <div className={darkMode ? "dark-mode" : "light-mode"}>
+      <div className="container">
+        <span style={{ color: darkMode ? "grey" : "yellow" }}>☀︎</span>
```
Comment (MuhammadAwaisGondal) [inline src/App.js]: It is better to remove the inline style and create a separate object for style for better code understanding

File: src/App.js [hunk 3]
```diff
@@ -1,13 +1,28 @@
...
+        <div className="switch-checkbox">
+          <label className="switch">
+            <input type="checkbox" onChange={() => setDarkMode(!darkMode)} />
+            <span className="slider round"> </span>
+          </label>
+        </div>
+        <span style={{ color: darkMode ? "#c96dfd" : "grey" }}>☽</span>
```
Comment (MuhammadAwaisGondal) [inline src/App.js]: Same here for the inline style

**Review by MuhammadAwaisGondal — COMMENTED** [review]
Overall the code is good and the functionality is working very well. I have left a few suggestion to improve the overall quality.

**Comment by NivedhaSrinivasan09** [comment]
The code looks fine to me and the feature works perfectly.  One suggestion would be the Welcome Message can be added directly.

**Review by MuhammadAwaisGondal — APPROVED** [review]
The changes look good now

**Review by NivedhaSrinivasan09 — APPROVED** [review]
Overall code looks good

**Pull request merged**